src/client-side-encryption/stateMachine.ts

```diff
@@ -56,90 +65,149 @@ const INSECURE_TLS_OPTIONS = [
 ];
 
 /**
- * @ignore
- * @callback StateMachine~executeCallback
- * @param {Error} [err] If present, indicates that the execute call failed with the given error
- * @param {object} [result] If present, is the result of executing the state machine.
- * @returns {void}
+ * Helper function for logging. Enabled by setting the environment flag MONGODB_CRYPT_DEBUG.
+ * @param msg - Anything you want to be logged.
  */
+export function debug(msg: unknown) {
```

Comment:
```suggestion
function debug(msg: unknown) {
```

Not used elsewhere right?